.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,112 @@
+name: "Bug Report"
+description: "File a bug report to help us improve"
+labels: [need/triage, kind/bug]
+body:
+- type: checkboxes
+  attributes:
+    label: Checklist
+    description: Please check off the following boxes before continuing to file a bug report!
+    options:
+      - label: This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
+        required: true
+      - label: This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
+        required: true
+      - label: This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=hint%2Fneeds-triaging&template=feature_request.yaml) instead.
+        required: true
+      - label: This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=hint%2Fneeds-triaging&template=enhancement.yaml) instead.
+        required: true
+      - label: I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
+        required: true
+      - label: I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
+        required: true
+      - label: I did not make any code changes to lotus.
+        required: true
+- type: dropdown 
+  id: component
+  validations:
+    required: true
+  attributes:
+    label: Lotus component
+    description: Please select the lotus component you are filing a bug for
+    options:
+    - lotus daemon
+    - lotus miner/lotus worker
+    - lotus client
+    - lotus market
+    - lotus JSON-RPC API
+    - Other
+- type: dropdown
+  id: area
+  validations: 
+    required: true
+  attributes:
+    label: Area
```

Comment:
This seems to overlap with the component section a bit. Personally, I'd just combine them.

Reply:
combined